lisp/tree-sitter-debug.el

```diff
@@ -57,11 +57,14 @@ This only takes effect if `tree-sitter-debug-jump-buttons' is non-nil."
   (push-mark (byte-to-position (cdr byte-range))
              tree-sitter-debug-highlight-jump-region))
 
-(defun tree-sitter-debug--display-node (named-p type start-byte end-byte depth)
+(defun tree-sitter-debug--display-node (named-p type start-byte end-byte depth field)
   "Display NODE that appears at the given DEPTH in the syntax tree."
   (when named-p
     (insert (make-string (* 2 depth) ?\ ))
-    (let ((node-text (format "%s:" type)))
+    (let* ((field-text (if field
+                           (format " (%s)" field)
+                         ""))
+           (node-text (format "%s%s:\n" type field-text)))
```

Comment:
Why an extra newline here? 🤔 

![2024-05-10 18 09 39](https://github.com/emacs-tree-sitter/elisp-tree-sitter/assets/8685505/556506ce-ccd6-4228-9aa3-6d4b2f6c3bf1)

Reply:
Oh! I think this is because I ported this patch to `master` from one based on a release, and I didn't realise that got added there.

Reply:
Fixed.